dashboard/db/migrate/20160430111030_add_kind_to_survey_results.rb

```diff
@@ -0,0 +1,12 @@
+class AddKindToSurveyResults < ActiveRecord::Migration
+  def change
+    add_column :survey_results, :kind, :string, index: true, :after => :user_id
```

Comment:
breville: Do you know why I'm not getting the index annotation in `survey_result.rb` or the index in `schema.rb`?